docs(contribute): add guide to run the document site (#1001)
eslint-stylistic/eslint-stylistic: docs/contribute/guide.md

> We use pnpm v10. If you are working on multiple projects with different versions of pnpm, it's recommended to enable [Corepack](https://github.com/nodejs/corepack) by running `corepack enable`.

> ### Building The Website
> 
> This site is built with [VitePress](https://vitepress.dev/).
> 
> To run the documentation site locally, execute `pnpm run docs` in the root folder.

Reviewer: ```suggestion
### Running Docs

This documentation site is built with [VitePress](https://vitepress.dev/).

To run it locally, execute `pnpm docs` at the root folder.
```
